frappe/database/database.py

```diff
@@ -982,7 +982,7 @@ def bulk_insert(self, doctype, fields, values):
 		for idx, value in enumerate(values):
 			insert_list.append(tuple(value))
 			if idx and (idx%10000 == 0 or idx < len(values)-1):
-				self.sql("""INSERT INTO `tab{doctype}` ({fields}) VALUES {values}""".format(
+				self.sql("""INSERT IGNORE INTO `tab{doctype}` ({fields}) VALUES {values}""".format(
```

Comment:
Adding `IGNORE` should be developers choice. It should not be added by default. Something like `frappe.db.bulk_insert('ToDo', fields, values, ignore_duplicates=True)`